src/Dialect/Krnl/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ add_onnx_mlir_library(OMKrnlOps
   KrnlOps.cpp
   KrnlTypes.cpp
   KrnlHelper.cpp
-  SimplifyKrnl.cpp
```

Comment:
I feel like there were some additional dependencies needed here because of SimplifyKrnl.cpp and perhaps some of the depends/libs below can be removed now? It was a while back when I touched this, so I am not sure if it's still pertinent.

Reply:
Also, yay!

Reply:
Thanks I was able  to remove one of the dependency.